lib/data_fetchers/git_fetcher.rb

```diff
@@ -38,18 +38,31 @@ def line_changes
   end
 
   def languages
-    languages = github.languages.map do |language, percent|
+    languages = github.languages
+      .merge(gitlab.languages) { |key, value1, value2| ((value1 + value2) / 2.0).round }
```

Comment:
Mirrorred repos are handled by adding them to the repo blacklist (see .env). Not a very elegant solution, but for now it's enough, as there aren't too many of them. For the cloning and updating of the local repos this needs to be handled anyway.